core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -385,4 +385,17 @@ package object config {
       .checkValue(v => v > 0 && v <= Int.MaxValue,
         s"The buffer size must be greater than 0 and less than ${Int.MaxValue}.")
       .createWithDefault(1024 * 1024)
+
+  private[spark] val UNROLL_MEMORY_CHECK_PERIOD =
+    ConfigBuilder("spark.storage.unrollMemoryCheckPeriod")
+      .doc("The memory check period is used to determine how often we should check whether "
```

Comment:
let's call `.internal()` here, to make them internal configs, as we don't expect users to tune it.